.github/workflows/ci.yml

```diff
@@ -10,9 +10,12 @@ on:
       - 'environment.yml'
 
 jobs:
-  test-dependency-img:
-    runs-on: ubuntu-latest
-    container: ghcr.io/svalinn/parastell-ci
+  runs-on: ubuntu-latest
+    test-dependency-img:
+      container: ghcr.io/svalinn/parastell-ci
```

Comment:
This change confuses me?

Reply:
We'll need to revert the first two lines
```suggestion
  test-dependency-img:
    runs-on: ubuntu-latest
    container: ghcr.io/svalinn/parastell-ci
```